clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/AwsProvider.groovy

```diff
@@ -58,7 +59,7 @@ class AwsProvider extends AgentSchedulerAware implements SearchableProvider, Eur
   final Collection<Agent> agents
   private Collection<HealthProvidingCachingAgent> healthAgents
 
-  AwsProvider(AccountCredentialsRepository accountCredentialsRepository, Collection<Agent> agents) {
+  AwsProvider(CredentialsRepository<? extends NetflixAmazonCredentials> accountCredentialsRepository, Collection<Agent> agents) {
```

Comment:
With the change above, it means `agents` aren't passed when instantiating the provider.